source/server/options_impl.cc

```diff
@@ -12,8 +12,16 @@
 #include "spdlog/spdlog.h"
 #include "tclap/CmdLine.h"
 
+#ifndef ENVOY_DEFAULT_MAX_STATS
+#define ENVOY_DEFAULT_MAX_STATS 16384
```

Comment:
nit: C++ const class values for both of these.

Reply:
This was so that it can be overridden at compile-time.  Do you want this define, and then a const class value using these #defines for the actual value?

Reply:
Ah OK. Please just add comment that this is for compile time override. We will need some docs on this in the bazel/README.md file.